src/test/javascript/spec/component/programming-exercise/programming-exercise-custom-build-plan.component.spec.ts

```
@@ -224,15 +215,14 @@ describe('ProgrammingExercise Custom Build Plan', () => {
 
     it('should accept editor for existing exercise', () => {
         comp.programmingExercise.id = 1;
-        const elementRef: ElementRef = new ElementRef(document.createElement('div'));
         comp.programmingExercise.buildConfig!.buildScript = 'buildscript';
-        const editor = new MonacoEditorComponent(mockThemeService, elementRef, renderer2, translateService);
+        const editor = TestBed.createComponent(MonacoEditorComponent).componentInstance;
         expect(comp.editor).toBeUndefined();
         comp.editor = editor;
         expect(comp.code).toBe('buildscript');
         expect(comp.editor).toBeDefined();
         comp.programmingExercise.buildConfig!.buildScript = undefined;
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)